hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/DownloadAndImportReplicator.java

```diff
@@ -70,9 +76,16 @@ public void replicate(ReplicationTask task) {
 
     LOG.info("Starting replication of container {} from {} using {}",
         containerID, sourceDatanodes, compression);
+    HddsVolume targetVolume = null;
 
     try {
-      HddsVolume targetVolume = containerImporter.chooseNextVolume();
+      targetVolume = containerImporter.chooseNextVolume();
+      targetVolume.incImportContainerCommitBytes(containerSize * 2);
```

Comment:
"containerSize * 2" is just checked during "containerImporter.chooseNextVolume()", there is no need to check it again here.